### PR TITLE
refactor the index of readthedocs

### PR DESCRIPTION
**Pull request by QuanluZhang**
resolve the issue #1827

### CODE REVIEW

File: docs/en_US/hpo_advanced.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Advanced Features
+=================
+
+..  toctree::
+    Enable Multi-phase <AdvancedFeature/MultiPhase>
```
Comment (xuehui1991) [inline docs/en_US/hpo_advanced.rst]: suggest putting the multi-phase in last.

File: docs/en_US/hyperparameter_tune.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
+######################
+Hyper-parameter Tuning
+######################
+
+..  toctree::
+    :maxdepth: 2
+
+    Write Trial <TrialExample/Trials>
```
Comment (xuehui1991) [inline docs/en_US/hyperparameter_tune.rst]: some of them are not just for HPO. I think the original name "tutorial" would be better.

And some of the doc about HPO doesn't include in.

File: docs/en_US/reference.rst [hunk 3]
```diff
@@ -2,12 +2,9 @@ References
 ==================
```
Comment (xuehui1991) [inline docs/en_US/reference.rst]: suggest changing "reference" to other name

Comment (xuehui1991) [reply docs/en_US/reference.rst]: And put it in the top.

File: docs/en_US/hyperparameter_tune.rst (hunk 2 above)
Comment (QuanluZhang) [reply docs/en_US/hyperparameter_tune.rst]: There is no tutorial now. Currently, other features does not use training service and WebUI. it is not proper to put them in reference. Do you have concrete suggestions?

could you specify "some of the doc"?

File: docs/en_US/reference.rst (hunk 3 above)
Comment (QuanluZhang) [reply docs/en_US/reference.rst]: could you suggestion for the name? cannot get your idea

File: docs/en_US/hyperparameter_tune.rst (hunk 2 above)
Comment (xuehui1991) [reply docs/en_US/hyperparameter_tune.rst]: 在一开始设计的时候 training service是一个和HPO平行的概念 而不是附属于HPO. 实时上我们network morphsim这种NAS的算法是支持跑在不同的training service上的。 即使新的NAS interface不一定支持跑在例如openpai这样的training service上，我们也不能说以后没有这种可能？

File: docs/en_US/reference.rst (hunk 3 above)
Comment (xuehui1991) [reply docs/en_US/reference.rst]: 如果是我 我觉得这些是用nni必须知道和了解的内容，所以他们的doc都放在了tutorial这个文件夹下面。 如果是我 我觉得可以把这个名字改成“tutorial.rst”

File: docs/en_US/hyperparameter_tune.rst (hunk 2 above)
Comment (QuanluZhang) [reply docs/en_US/hyperparameter_tune.rst]: yes, you are right. it is not only considering this specific part, we should consider the whole structure. we have four parts, but HPO is the most similar to the previous tutorial, thus we decide to explicitly show these four capabilities like #1827

File: docs/en_US/hpo_advanced.rst (hunk 1 above)
Comment (Ronniechase12) [reply docs/en_US/hpo_advanced.rst:5]: @jodiemurphy13

**Pull request merged**